test/cli/premium_test.py

```diff
@@ -26,8 +26,7 @@ def __copy_cppcheck_premium(tmpdir):
                 {
                     "addons": [],
                     "productName": "NAME",
-                    "about": "NAME",
-                    "safety": true
+                    "about": "NAME"
```

Comment:
This test was based on the behavior that is loaded `cppcheck.cfg` first and being overriden by the CLI. But that is not how it was initially implemented. That behavior changed when the double loading was introduced in #5760.